docs/EN_US/HPCCSystemAdmin/SA-Mods/DAFtslave.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE section PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect2 id="SA_SupportUtilities">
+  <title>Support Utilities</title>
+
+  <para>There are a few additional components which are neither system servers
+  or client interfaces but nonetheless important in supporting HPCC
+  tasks.</para>
+
+  <sect3 id="DAFilesrv">
+    <title>dafilesrv</title>
+
+    <para>The dafilesrv is a daemon that runs on system servers or physical
+    locations on the HPCC system. This daemon enables the HPCC components to
+    have file access to physical file locations. File access could be
+    spraying, despraying, copying, etc. It is also used to enable file
+    security.</para>
```

Comment:
>It is also used to enable file security

That could be misleading. Or need to be clear that talking about physical vs logical security.
Dafilesrv can be configured to allow SSL connections only, which secures remote access to physical files.